### DISCUSSION TITLE
Data importer for a separate account

### DISCUSSION THREAD

**Discussion by jquiros2** [firefly-iii/firefly-iii#7751] [Firefly III]
Hi!  I am using firefly III for myself quite well, with the importer.
I created an account for someone else, and am wondering how to import their data.

I did not find info about this, so my question is:

Can I import data for a different account?  If so, how?

**Comment by JC5** [comment]
How is the data importer configured right now? To use a personal access token?

If you configure it with only a Firefly III URL it will ask for a client ID. That ID can be generated by the different account and then used.

**Comment by jquiros2** [comment]
Thank you!

Since I am running in docker, in principle I could create a second data-importer container, with the other auth code in a separate .importer.env (.importer2.env FIREFLY_III_ACCESS_TOKEN) file, mapped to a different internal port/external url.

Would I also use "FIREFLY_III_CLIENT_ID" in this case?

Will try this soon and post back here when done.

**Reply by JC5** [reply]
If it's all local, you could even use two tokens.
